products/acsf/features/php-administrator.feature

```diff
@@ -39,11 +42,15 @@ Feature: PHP
     Then I should not see "Evaluate PHP in pattern"
 
   @api
-  Scenario: Backup and Migrate Import Form
+  Scenario: Backup and Migrate Import and Restore Forms
     Given I am logged in as a user with the "administrator" role
     And the "backup_migrate" module is enabled
     And I am on "admin/config/system/backup_migrate/settings/import"
     Then I should see "This form is disabled"
+    Given I am on "admin/config/system/backup_migrate/restore"
+    Then I should see "Access denied"
+    And I should see "You are not authorized to access this page."
```

Comment:
On ACSF, ie. with SAML, the error message is: "Web Login Service - Unable to Respond."  Let me know if we want these to run only on non-SAML enabled sites.

Reply:
Hm, the session should persist. It shouldn't prompt for Weblogin. Let me check it on ACSF.

Reply:
Blerh.  Thank you for the session reminder.  That was my fault.  Here's what I see with a non user1 admin account:

![screen shot 2018-02-14 at 2 50 18 pm](https://user-images.githubusercontent.com/4072139/36232381-6dde79bc-1196-11e8-9fb0-2a71a5f95e3c.png)

Reply:
Well isn't *that* interesting

h2. Local
![2018-02-14-15551-screenshot](https://user-images.githubusercontent.com/821106/36234498-337e39a6-11a0-11e8-923b-29d3e442744b.jpg)

h2. ACSF
![2018-02-14-16022-screenshot](https://user-images.githubusercontent.com/821106/36234512-3d13a14a-11a0-11e8-8fce-d8543956de36.jpg)

Reply:
Oh, I figured out why. You didn't save the form at `admin/people/permissions` to register the new revoked permission (in fairness, I didn't include that instruction in the steps to test). So as an admin user on the ACSF site, you were getting the response that User 1 gets, which is the implementation of `paranoia_risky_forms()`, rather than the implementation of `hook_paranoia_hide_permissions()`.

Try again on https://sites207f-dev.cardinalsites.stanford.edu/

Reply:
I tried re-saving admin/people/permissions, just in case, but I'm afraid, the scenario still fails and shows in the GUI:

![screen shot 2018-02-14 at 5 13 11 pm](https://user-images.githubusercontent.com/4072139/36236326-694bebc8-11aa-11e8-8cc0-8fc504565a64.png)